### PR TITLE
Add tf2pytorch wrapper in transformers tool

### PR DESCRIPTION
**Pull request by wangyems**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/transformers/convert_tf_models_to_pytorch.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    else:
+        model_categroy_name = TFMODELS[model_name][2]
+        module = __import__("transformers", fromlist=[model_categroy_name])
+        model_categroy = getattr(module, model_categroy_name)
+        init_model = model_categroy(config)
+    return config, init_model
+
+
+def convert_tf_checkpoint_to_pytorch(model_name, config, init_model, tf_checkpoint_path):
+    load_tf_weight_func_name = "load_tf_weights_in_" + TFMODELS[model_name][0]
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/convert_tf_models_to_pytorch.py:112]: Note that bert model trained by TF 2.* use load_tf2_weights_in_bert as in 
https://github.com/huggingface/transformers/blob/77cd0e13d2d09f60d2f6d8fb8b08f493d7ca51fe/src/transformers/convert_bert_original_tf2_checkpoint_to_pytorch.py

File: onnxruntime/python/tools/transformers/onnx_exporter.py [hunk 2]
```diff
@@ -229,6 +229,34 @@ def load_pretrained_model(model_name, config, cache_dir, custom_model_class, is_
...
+    config = AutoConfig.from_pretrained(model_name, cache_dir=cache_dir)
+
+    model = load_pretrained_model(model_name, config=config, cache_dir=cache_dir, custom_model_class=model_class, is_tf_model=True)
+
+    return config, model
+
+
+# For test only
+def load_pt_model_from_tf(model_name):
+    # Note that we could get pt model from tf, but model source and its structure in this case is different from directy using
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/onnx_exporter.py:252]: typo: directly

File: onnxruntime/python/tools/transformers/onnx_exporter.py [hunk 3]
```diff
@@ -267,11 +295,8 @@ def export_onnx_model_from_pt(model_name, opset_version, use_external_data_forma
                               input_names, use_gpu, precision, optimize_onnx, validate_onnx, use_raw_attention_mask, overwrite,
                               model_fusion_statistics):
 
-    config = AutoConfig.from_pretrained(model_name, cache_dir=cache_dir)
-    if hasattr(config, 'return_dict'):
-        config.return_dict = False
-
-    model = load_pretrained_model(model_name, config=config, cache_dir=cache_dir, custom_model_class=model_class)
+    config, model = load_pt_model(model_name, model_class, cache_dir)
+    # config, model = load_pt_model_from_tf(model_name)
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/onnx_exporter.py:299]: Could you add a unit test for load_pt_model_from_tf?

File: onnxruntime/python/tools/transformers/convert_tf_models_to_pytorch.py (hunk 1 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/convert_tf_models_to_pytorch.py:112]: added

File: onnxruntime/python/tools/transformers/onnx_exporter.py (hunk 3 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/onnx_exporter.py:299]: added a test for tf2pt_pipeline

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**